### ISSUE TITLE
0.8.6 Unable to export folder structure to compendium

### ISSUE THREAD

**Issue by KingN0thing** [earlSt1/vtt-compendium-folders#77]
I get this console error when trying to export folder structure to a compendium:
```
Uncaught (in promise) TypeError: Right-hand side of 'instanceof' is not an object
    at exportSingleFolderToCompendium (compendium-folders.js:1984)
    at async recursivelyExportFolders (compendium-folders.js:1944)
    at async callback (compendium-folders.js:1878)
exportSingleFolderToCompendium @ compendium-folders.js:1984
```
I am using Foundry 0.8.6, all modules are up-to-date. I tried disabling most of my other modules thinking there might be a conflict, but the error is still occuring.

Thank you

**Comment by earlSt1** [comment]
Hi, i actually fixed this earlier this morning. Havent pushed a release yet as i had some other stuff to look into. 

For the time being you can use the test version which just includes this fix, then when i push an update for the module later it will automatically update. Just uninstall your current version of compendium folders and install this one using the manifest input

https://raw.githubusercontent.com/earlSt1/vtt-compendium-folders/082-update/test/module.json

**Comment by earlSt1** [comment]
Fix included in v2.3.2
